source/change-streams/change-streams.rst

```diff
@@ -137,7 +137,21 @@ If an aggregate command with a ``$changeStream`` stage completes successfully, t
      * MUST NOT err when they encounter a new `operationType`. Unknown `operationType`
      * values may be represented by "unknown" or the literal string value.
      */
-    operationType: "insert" | "update" | "replace" | "delete" | "invalidate" | "drop" | "dropDatabase" | "rename";
+    operationType: "insert" 
+                  | "update" 
+                  | "replace" 
+                  | "delete" 
+                  | "invalidate" 
+                  | "drop" 
+                  | "dropDatabase" 
+                  | "rename" 
+                  | "createIndexes"
+                  | "dropIndexes" 
+                  | "modify"
+                  | "create" 
+                  | "shardCollection" 
+                  | "refineCollectionShardKey" 
+                  | "reshardCollection";
```

Comment:
What about `migrateLastChunkFromShard`?

Reply:
`migrateLastChunkFromShard` is guarded behind `showSystemEvents`, not `showExpandedEvents`. This flag is undocumented.

I think that Go may have implemented this change event for atlas to use internally but other drivers shouldn't need to expose this event.